src/main/scala/com/typesafe/sbt/packager/jdkpackager/JDKPackagerPlugin.scala

```diff
@@ -5,82 +5,78 @@ import com.typesafe.sbt.packager.Keys._
 import com.typesafe.sbt.packager.SettingsHelper
 import com.typesafe.sbt.packager.archetypes.JavaAppPackaging
 import com.typesafe.sbt.packager.archetypes.jar.ClasspathJarPlugin
+import com.typesafe.sbt.packager.jdkpackager.JDKPackagerKeys.JavaFXToolkit
 import sbt.Keys._
 import sbt._
 import SbtNativePackager.Universal
+
 /**
  * Package format via Oracle's packaging tool bundled with JDK 8.
+ *
  * @author <a href="mailto:fitch@datamininglab.com">Simeon H.K. Fitch</a>
  * @since 2/11/15
  */
 object JDKPackagerPlugin extends AutoPlugin {
 
-  object autoImport extends JDKPackagerKeys {
-    val JDKPackager = config("jdkPackager") extend Universal
-  }
+  object autoImport extends JDKPackagerKeys
```

Comment:
The `autoImport` should also contain the `JDKPackagerToolkit` so the user doesn't have to import it manually.